features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende geboorte gegevens
+      | naam                                      | waarde   |
+      | datum (01.03.10)                          | 19860401 |
+      | land.code                                 | 6030     |
+      | plaats.code                               | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999994086                                |
+      | fields              | code                                     |
+      Dan bevat de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En bevat de response een invalidParams[0] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.land.code  |
+      En bevat de response een invalidParams[1] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.plaats.code  |
+
+  Rule: De response bevat alleen de properties die in de fields parameter gevraagd en indien relevant de volgende properties :
+        De volgende velden worden geleverd ongeacht of de fields parameter is gebruikt en ongeacht de eventueel gevraagde velden in de fields parameter:
+        - inOnderzoek wordt meegegeven wanneer gevraagde corresponderende velden in onderzoek zijn. Zie in_onderzoek.feature voor uitleg wanneer attributen in onderzoek zijn.
+        - geheimhoudingPersoonsgegevens wordt altijd meegegeven wanneer deze de waarde true heeft.
+        - opschortingBijhouding reden en datum worden altijd meegegeven wanneer deze een waarde hebben.
+
+
+    Scenario: Slechts één enkel attribuut wordt gevraagd
```

Comment:
ik begrijp niet hoe de volgende 4 scenario's illustreren dat bepaalde gegevens ook worden geleverd wanneer daar niet om gevraagd is.

Reply:
Ik heb die rule gesplitst in alleen leveren van properties die met Fields gevraagd worden en een nieuwe rulke dat velden geleverd worden ongeacht of ze met fields worden gevraagd.